common/src/stack/command/stack/commands/__init__.py

```diff
@@ -1893,49 +1893,49 @@ def endOutput(self, header=[], padChar='-', trimOwner=False, trimHeader=False):
 				for i in range(0, rows):
```

Comment:
While you are modifying this function, you should change `range(0, ...blah...` to remove the first parameter (it defaults to 0): `for i in range(rows):`

Reply:
How much do we generally like to scope creep on tickets? I probably shouldn't have renamed the variable names that overloaded builtins, but it seemed too egregious to leave in.

I'm used to these kinds of overarching "fix the linting errors" ending up in some ticket, or set of tickets, to sweep the entire file/codebase so the changes are consistent throughout and reduce diff noise unrelated to the original ticket. I'd argue that these would fall in this category and maybe I'll even move the overridden builtins under that change set. Thoughts @bsanders @caladd ?

Reply:
I'd recommend breaking this up into two bugfix branches, one where you simply fix the pretty print indent (and the tests that fail with the new spaces). And a second branch to clean up the `Command::endOutput` function variable names and other code.